vizro-core/tests/unit/vizro/models/_components/form/test_slider.py

```diff
@@ -0,0 +1,142 @@
+"""Unit tests for hyphen.models.slider."""
+import json
+
+import plotly
+import pytest
+from dash import dcc
+from pydantic import ValidationError
+
+import vizro.models as vm
+from vizro.actions import export_data
+from vizro.models._action._actions_chain import ActionsChain
+
+
+class TestSliderInstantiation:
+    """Tests model instantiation."""
+
+    def test_create_default_slider(self):
+        slider = vm.Slider()
+
+        assert hasattr(slider, "id")
+        assert slider.type == "slider"
+        assert slider.min is None
+        assert slider.max is None
+        assert slider.step is None
+        assert slider.marks is None
+        assert slider.value is None
+        assert slider.title is None
+        assert slider.actions == []
+
+    @pytest.mark.parametrize(
+        "min, max, step, marks, value, title",
+        [
+            (0, 10, 1, {}, 3, "Test text"),
+            (100, 200, 50, {}, 100, """## Test header"""),
+            (1.23, 6.78, 0.01, {}, None, 1.23),
+            (0, 10, None, {str(i): i for i in range(0, 10, 1)}, 5, ""),  # add marks
+        ],
+    )
+    def test_create_slider_with_optional(  # noqa
+        self, min, max, step, marks, value, title
+    ):
+        slider = vm.Slider(
+            min=min, max=max, step=step, marks=marks, value=value, title=title
+        )
+
+        assert slider.min == min
+        assert slider.max == max
+        assert slider.step == step
+        assert slider.marks == marks
+        assert slider.value == value
+        assert slider.title == str(title)
+
+    def test_invalid_reverse_slider(self):
+        with pytest.raises(
+            ValidationError,
+            match="Minimum value of slider is required to be smaller than maximum value.",
+        ):
+            vm.Slider(min=10, max=0)
+
+    def test_set_default_marks(self):
+        slider = vm.Slider(min=0, max=10, step=1)
+        assert slider.marks == {}
+
+    def test_create_slider_with_action(self):
+        slider = vm.Slider(actions=[vm.Action(function=export_data())])
+        slider_tac = slider.actions[0]
+        assert hasattr(slider, "id")
+        assert slider.type == "slider"
+        assert len(slider.actions) == 1
+        assert isinstance(slider_tac, ActionsChain)
+        assert slider_tac.trigger.component_property == "value"
+
+
+class TestBuildMethod:
+    def test_slider_build(self):
```

Comment:
I think I would take a more sophisticated example in here or have two tests, one which you already have (the basic default) and another one where you test the build of a slider with min, max, marks, title?